src/TimerOutputs.jl

```diff
@@ -1,7 +1,7 @@
 module TimerOutputs
 
 import Base: show, time_ns
-export TimerOutput, @timeit, @timeit_debug, reset_timer!, print_timer, timeit
+export TimerOutput, @timeit, @timeit_debug, reset_timer!, print_timer, timeit, enable!, disable!
```

Comment:
```suggestion
export TimerOutput, @timeit, @timeit_debug, reset_timer!, print_timer, timeit, enable_timer!, disable_timer!
```
perhaps? Fits better with the other exported functions.